xml/operating_system.xml

```diff
@@ -89,6 +90,34 @@
     <param pos="0" name="os.family" value="Windows"/>
     <param pos="1" name="os.product"/>
   </fingerprint>
+  <fingerprint pattern="^(?i:(?:Microsoft )?Windows 6.1)$">
```

Comment:
It might be worth including `(?:\sNT)?` between `Windows` and the version number, as that is another way these versions can be displayed.